src/Plutarch/Extra/AssetClass.hs

```diff
@@ -3,6 +3,7 @@
 {-# LANGUAGE TypeApplications #-}
 {-# LANGUAGE TypeFamilies #-}
 {-# LANGUAGE UndecidableInstances #-}
+{-# OPTIONS_GHC -Wno-missing-import-lists #-}
```

Comment:
A comment explaining why this is off would be good I think, for future reference if nothing else.